src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -106,6 +107,14 @@ public void stop() {
     spinWheels(0);
   }
 
+  public Command pullTimed(double seconds, boolean isConeMode){
+    return new StartEndCommand(this::pull, this::stop, this).withTimeout(seconds).beforeStarting(isConeMode ? this::setConeMode : this::setCubeMode);
+  }
+
+  public Command pushTimed(double seconds, boolean isConeMode){
```

Comment:
Change isConeMode parameter name so it isn't confused with the class variable

Reply:
Changed!